plugins/module_utils/runner.py

```diff
@@ -148,6 +149,9 @@ def perform_action(module, definition: Dict[str, Any]) -> Tuple[bool, Dict[str,
             except Exception as e:
                 raise FlightctlException(f"Failed to create resource: {e}") from e
 
+    if result is None:
+        raise FlightctlException("No result returned from operation.")
```

Comment:
I'm not sure, but I don't think this check is necessary because if something fails it should raise an exception.